ERCS/erc-7574.md

```diff
@@ -0,0 +1,191 @@
+   "external_url":"https://api.kor.credential/metadata/1/1",
+   "home_url":"https://app.kor.credential/token/1",
+   "image_url":"https://www.poap.xyz/events/badges/ethdenver-19.png",
+   "name":" KOREA KYC",
+   "attributes" : { ... },
+   "Iusser" : { ...},
+   "credentialNumber" : { ... },
+ }
+```
+
+We have defined a structure to represent VerifiedPresentation. It includes the user's wallet address, issuer, user name, and Credential Number
+
+```jsx
+struct VerifiedPresentation {
+address userAddr;
+string issuer;
+string user;
+uint CredentialNumber;
+}
+```
+
+Following that, here is the contract that verifies a user's Credential proof and issues SBTs:
+
+
+### Contract Interface
+
+
+In this scenario, a **`verify`** function is created using Zokrates. Upon successful verification, the generated **`verify`** function calls the SBT issuance contract, enabling the entire process.
+
+```
+/ SPDX-License-Identifier: CC0-1.0
+pragma solidity^0.8.0;interface IERC5192 {
+
+/// @ Function to verify the user's proof.
+/// @ Generated through Zokrates, required parameters upon completion of generation are proof, input, and numofCred.
+/// @ Other information such as issuer/holder is optional.
+function verify(Proof memory proof, uint[2] memory input, uint numOfCred) returns ();
+
+/// @ Only those who have completed verifyProof can obtain the opportunity to issue SBT.
+/// @ Only the service provider can call the createSBT function.
+/// @ Issues SBT to authenticated users, with the verified credential Number and Issuer being input for tokenURI.
+function createSBT(address user, String memory tokenURI) public onlyOwner returns(uint256) public onlyWoner returns(uint256))
+
+/// @ Only the service provider can call the createSBT function.
+/// @ Issuing an SBT allows cancellation effects by severing the user's tokenURI, such as when the credential is revoked.
+function updateTokenURI(address user, string memory tokenURI) public onlyOwner
+
+/// @ Added cannotTransfer functionality to prevent the transfer of SBT to others.
+function safeTransferFrom(adress _from, address _to, uint256 _tokenId) public virtual oveerid
+
+}
+```
+
+Through this process, obtaining SBT is the only means of authentication, and the possession of SBT subsequently serves as a method for identity verification.
+
+
+## Rationale
+
+<!--
+  The rationale fleshes out the specification by describing what motivated the design and why particular design decisions were made. It should describe alternate designs that were considered and related work, e.g. how the feature is supported in other languages.
+
+  The current placeholder is acceptable for a draft.
+
+  TODO: Remove this comment before submitting
+-->
+
+This is a system that verifies the basic DID signature by integrating the Ethereum extension program Zokrates. Following the verification of the DID signature, it proceeds with issuing SBT, allowing the validation of a user's identity information.
```

Comment:
Your Rationale section is a bit light. You want to explain the reasoning behind technical choices you made within the ERC itself.

My pet analogy for this is:

> Motivation: "why do we need a shed?"
> Rationale: "why is the shed red?"